docs/data/charts/sankey/SankeyNodeSorting.js

```diff
@@ -32,14 +40,37 @@ export default function SankeyNodeSorting() {
     >
       <div>
         <Typography variant="h6" gutterBottom>
-          Default Node Order
+          auto (default)
+        </Typography>
+        <SankeyChart
+          height={300}
+          series={{
+            data,
+            nodeOptions: {
+              sort: 'auto',
+            },
+          }}
+        />
+      </div>
+
+      <div>
+        <Typography variant="h6" gutterBottom>
+          fixed
```

Comment:
Fixed and auto are almost identical. Should we change the data so they're more dissimilar and it's easier to tell them apart?